vizro-core/docs/pages/examples/examples.md

```diff
@@ -13,14 +13,14 @@ The dashboard launches with a home page that offers four other pages:
 * Continent summary: Summarizes the main findings for each continent.
 
 
-You can find the code for each of the charts, for each page of the dashboard, in the `examples` folder of the `vizro-core` package, within [Vizro's GitHub repository](https://github.com/mckinsey/vizro). The code is available as a [`.py` file](https://github.com/mckinsey/vizro/blob/main/vizro-core/examples/demo/app.py) or as a [Jupyter Notebook](https://github.com/mckinsey/vizro/tree/main/vizro-core/examples/demo/jupyter_version).
+You can find the code for each of the charts, for each page of the dashboard on our [Hugging Face gapminder demo space](https://huggingface.co/spaces/vizro/gapminder). The code is available as a [`.py` file](https://huggingface.co/spaces/vizro/gapminder/blob/main/app.py).
```

Comment:
```suggestion
You can find the code for each of the charts, for each page of the dashboard on our [Hugging Face gapminder demo space](https://huggingface.co/spaces/vizro/demo_gapminder). The code is available as a [`.py` file](https://huggingface.co/spaces/vizro/demo_gapminder/blob/main/app.py).
```

Reply:
Could you change it to dashes instead of underscores on Hugging Face? To be in line how we convert paths and in general guidelines for URLs: https://stackoverflow.com/questions/119312/urls-dash-vs-underscore 

I'll change it here already :)